pyproject.toml

```diff
@@ -1,2 +1,58 @@
 [tool.black]
 target-version = ['py310']
+
+[tool.poetry]
```

Comment:
Not yet.  https://github.com/python-poetry/poetry/issues/3332
This might be a good enough reason to look at using another build-system. It's unclear to me what other packages look at the `[project]` section, but it feels like it's potentially going to be an issue at some point.